Islandora/jsonld
# Convert to GitHub actions

**GitHub Issue**: (https://github.com/Islandora/documentation/issues/1798)

Big credit goes to @seth-shaw-unlv  for helping me fix the tests with phpunit 9

# What does this Pull Request do?
Converts to github actions
updates the badge
removes travis

# What's new?
* new CI runner

# How should this be tested?
* make sure the tests pass (i don't think they will until this PR is merged - but you can see them running over on the branch itself)

# Interested parties
@Islandora/8-x-committers @seth-shaw-unlv
